app/messaging/save-to-database.js

```diff
@@ -1,29 +1,23 @@
+const { format, parse } = require('date-fns')
```

Comment:
Did use a dedicate NPM package ([date-fns](https://www.npmjs.com/package/date-fns)) to fix the issue with inputting the `requestedDate` on the AHWP stub form in the correct UK format. Found this to be quite a useful/popular package as it simplifies handling dates in JavaScript, and required much less code compared to the methods I was trying out before (e.g. `toLocaleDateString('en-GB')`, `getDate`, `getMonth`, `getFullYear` etc.). Not sure if it would have been preferred to solve the issue without installing an NPM package though?

Reply:
**UPDATE:** Went over how to solve the issue _without_ using an NPM package, even less code needed so will use that solution instead.